### ISSUE TITLE
A node cannot be added to trashed parent

### ISSUE THREAD

**Issue by lloydie2009** [yadayada/acd_cli#351]
Hello,

I'm seeing a new issue I've not seen before and wondered if anyone could translate what it means.

There's doesn't seen to be any other mention of this online.

A node cannot be added to trashed parent

16-06-20 20:45:40.021 [WARNING] [acdcli.api.backoff_req] - Waiting 7.537852s because of error(s).
16-06-20 20:46:58.643 [ERROR] [acd_cli] - Uploading "9WQZReR0ERWJpuYyJqE,MsZQ1ltAE3xEcZ,QCUfiF1DXKi7T7Kep-kVzHBAGH-S8lL7" failed. RequestError: 400, {"logref":"3dcdc676-3717-11e6-8452-131bb21bc138","message":"A node cannot be added to trashed parent","code":""}

Thanks for any help

**Comment by yadayada** [comment]
That happens if you try to upload a file into a folder that is trashed in the Drive, but not trashed in the local cache.

Please see if it still occurs after syncing.

**Comment by lloydie2009** [comment]
Yes, my upload process fully includes ACD_CLI sync before the upload and I still see this error.

**Comment by yadayada** [comment]
Sorry, I was assuming you had created a folder (and cached that folder info), trashed it and created a new folder of the same name as the trashed folder (not using acd_cli) and were trying to upload there.

If you are, in fact, trying to upload into a trashed folder, I regret that that is not possible.

**Comment by lloydie2009** [comment]
Neither of the above are relavant to my setup.

I'm simply trying to upload files that don't exist. I suppose theoretically the folder names could have existed at so.e point preciously, however I have ensure my ACD trash is completely empty, and I still get the same error. Its not all files, but a good percentage of them.

Thanks
